### PR TITLE
siren/police emoji dividers

### PR DESCRIPTION
**Pull request by haliphax**
## Description

Some dividers to kick off Hacktoberfest! #297

### CODE REVIEW

**Comment by haliphax** [comment]
These only count October 1-31, so I guess I'll close this for now and reopen it in a couple of days. Cheers!

**Review by RDIL — CHANGES_REQUESTED** [review]
Unfortunately, at this time we aren't accepting dividers made up of more than one unique character.  If you were to change it to fit that criteria, I can merge, but right now I can't.  

Thank you!

File: area4/dividers.txt [hunk 1]
```diff
@@ -283,4 +283,5 @@ $$$$$$$$$$$$
 øøøøøøøøøøøø
 ØØØØØØØØØØØØ
 🔭🔭🔭🔭🔭🔭🔭🔭🔭🔭🔭🔭
->>>>>>>>>>>>
```
[the remark this answers is not in the document]
Comment (RDIL) [reply area4/dividers.txt:286]: Whoops, I thought it was a conflict marker!  Thanks for noticing, I'll fix that.

[the remark this answers is not in the document]
Comment (haliphax) [reply area4/dividers.txt:286]: Ha! I did, too. My bad.

**Pull request merged**